app/src/main/java/com/android/unio/ui/event/EventCreation.kt

```diff
@@ -1,20 +1,379 @@
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material.icons.filled.AccessTime
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.DateRange
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.DatePicker
+import androidx.compose.material3.DatePickerDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedTextField
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.TimeInput
+import androidx.compose.material3.rememberDatePickerState
+import androidx.compose.material3.rememberTimePickerState
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.input.pointer.PointerEventPass
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
 import com.android.unio.R
 import com.android.unio.model.strings.test_tags.EventCreationTestTags
+import com.android.unio.ui.navigation.NavigationAction
+import com.android.unio.ui.theme.AppTypography
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
 
 @SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
 @Composable
-fun EventCreationScreen() {
+fun EventCreationScreen(navigationAction: NavigationAction) {
   val context = LocalContext.current
-  Scaffold(modifier = Modifier.testTag(EventCreationTestTags.SCREEN)) {
-    Text(context.getString(R.string.event_creation_screen))
+  var name by remember { mutableStateOf("") }
+  var shortDescription by remember { mutableStateOf("") }
+  var longDescription by remember { mutableStateOf("") }
+
+  Scaffold(
+      modifier = Modifier.testTag(EventCreationTestTags.SCREEN),
+      content = { padding ->
+        Column(
+            modifier =
+                Modifier.padding(padding)
+                    .padding(horizontal = 20.dp, vertical = 20.dp)
+                    .fillMaxWidth()
+                    .verticalScroll(rememberScrollState()),
```

Comment:
Maybe you should extract `rememberScrollState`, I am not sure but I think it creates a remember each refresh, since it isn't stored in a variable.